compiler/src/dotty/tools/dotc/transform/ElimRepeated.scala

```diff
@@ -266,7 +267,7 @@ class ElimRepeated extends MiniPhase with InfoTransformer { thisPhase =>
    *  The solution is to add a method that converts its argument from `Array[? <: T]` to `Seq[T]` and
    *  forwards it to the original method.
    */
-  private def addVarArgsForwarder(original: Symbol, isBridge: Boolean, hasAnnotation: Boolean)(using Context): Unit =
+  private def addVarArgsForwarder(original: Symbol, isBridge: Boolean, hasAnnotation: Boolean, parentHasAnnotation: Boolean)(using Context): Unit =
```

Comment:
Can you add a description for this parameter to the documentation comment above? (looks like hasAnnotation is also missing doc)

Reply:
done!